pcsx2-qt/Settings/MemoryCardConvertDialog.cpp

```diff
@@ -11,6 +11,8 @@
 #include "common/Path.h"
 #include "common/StringUtil.h"
 
+#include <unistd.h>
```

Comment:
This is Unix/POSIX only. Are you able to implement such a feature for Windows as well?

Reply:
I will see what I can do!

Reply:
> This is Unix/POSIX only. Are you able to implement such a feature for Windows as well?

Hey, I pushed some changes